examples/bioasq9b.py

```diff
@@ -0,0 +1,232 @@
+# coding=utf-8
+# Copyright 2020 The HuggingFace Datasets Authors and the current dataset script contributor.
```

Comment:
do we want to have people add their name here in place of `the current dataset script contributor`? or have people append their name if they make a PR?